setup.py

```diff
@@ -1,150 +1,23 @@
-        for list_key in 'use_scm_version', 'zip_safe':
-            try:
-                opt[list_key] = cfg_val_to_primitive(opt[list_key])
-            except KeyError:
-                pass
-        for list_key in 'scripts', 'install_requires', 'setup_requires':
-            try:
-                opt[list_key] = cfg_val_to_list(opt[list_key])
-            except KeyError:
-                pass
-        try:
-            opt['package_dir'] = cfg_val_to_dict(opt['package_dir'])
-        except KeyError:
-            pass
-        try:
-            opt_package_data = dict(cfg.items('options.package_data'))
-            if not opt_package_data.get('', '').strip():
-                opt_package_data[''] = opt_package_data['*']
-                del opt_package_data['*']
-        except (KeyError, NoSectionError):
-            opt_package_data = {}
-        try:
-            opt_extras_require = dict(cfg.items('options.extras_require'))
-            opt['extras_require'] = {}
-            for k, v in opt_extras_require.items():
-                opt['extras_require'][k] = cfg_val_to_list(v)
-        except NoSectionError:
-            pass
-        opt['package_data'] = {}
-        for k, v in opt_package_data.items():
-            opt['package_data'][k] = cfg_val_to_list(v)
-        cur_pkgs = opt.get('packages', '').strip()
-        if '\n' in cur_pkgs:
-            opt['packages'] = cfg_val_to_list(opt['packages'])
-        elif cur_pkgs.startswith('find:'):
-            opt_packages_find = dict(cfg.items('options.packages.find'))
-            opt['packages'] = setuptools.find_packages(**opt_packages_find)
-        return {'metadata': md, 'options': opt}
-
-
-setup_params = {}
-declarative_setup_params = read_configuration('setup.cfg')
-
-# Patch incorrectly decoded package_dir option
-# ``egg_info`` demands native strings failing with unicode under Python 2
-# Ref https://github.com/pypa/setuptools/issues/1136
-declarative_setup_params['options']['package_dir'] = {
-    str(k): str(v)
-    for k, v in declarative_setup_params['options']['package_dir'].items()
-}
-
-setup_params = dict(setup_params, **declarative_setup_params['metadata'])
-setup_params = dict(setup_params, **declarative_setup_params['options'])
-
-
 def cut_local_version_on_upload(version):
+    """Generate a PEP440 local version if uploading to PyPI."""
     import os
     import setuptools_scm.version  # only present during setup time
-    IS_PYPI_UPLOAD = os.getenv('PYPI_UPLOAD') == 'true'
+
+    IS_PYPI_UPLOAD = os.getenv('PYPI_UPLOAD') == 'true'  # set in tox.ini
     return (
-        '' if IS_PYPI_UPLOAD
+        ""
+        if IS_PYPI_UPLOAD
         else setuptools_scm.version.get_local_node_and_date(version)
     )
 
-
+setup_params = {}
 setup_params['use_scm_version'] = {
     'local_scheme': cut_local_version_on_upload,
 }
 
 
-__name__ == '__main__' and setuptools.setup(**setup_params)
+__name__ == "__main__" and setuptools.setup(**setup_params)
```

Comment:
Why did you change the style?

Reply:
Leftovers, fixing them. You can blame "black PTSD" for using double quotes. BTW, You should consider activating black on ansible-lint so we do not have to waste time discussing about formatting.

Reply:
No, black is still bad.